stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/client_endpoint.bal

```diff
@@ -27,18 +43,19 @@ public struct TargetService {
     SSL ssl;
 }
 
-@Description { value:"ClientEndpointConfiguration struct represents options to be used for HTTP client invocation" }
+@Description {value:"ClientEndpointConfiguration struct represents options to be used for HTTP client invocation"}
 @Field {value:"endpointTimeout: Endpoint timeout value in millisecond"}
 @Field {value:"keepAlive: Specifies whether to reuse a connection for multiple requests"}
 @Field {value:"transferEncoding: The types of encoding applied to the request"}
 @Field {value:"chunking: The chunking behaviour of the request"}
 @Field {value:"httpVersion: The HTTP version understood by the client"}
 @Field {value:"forwarded: The choice of setting forwarded/x-forwarded header"}
 @Field {value:"followRedirects: Redirect related options"}
-@Field {value:"retry: Retry related options"}
+@Field {value:"retryConfig: Retry related options"}
```

Comment:
This was recently renamed as retry. Any reason of renaming it back?

Reply:
No, this was by mistake.. I renamed it back to `retry`.. Will change this field as well.